packages/ava/src/data/statistics/index.ts

```diff
@@ -251,9 +251,9 @@ export function missing(value: unknown[]): number {
  * Calculate the counts of each distinct value in the array.
  * @param value - The array to process
  */
-export function valueMap(value: (string | number)[]): Record<string, number> {
+export function valueMap(value: unknown[]): Record<string, number> {
   const data: Record<string | number, number> = {};
-  value.forEach((v) => {
+  value.forEach((v: string | number) => {
```

Comment:
is there any ts error without type assertions ?     unknown => string | number

Reply:
fixed